### DISCUSSION TITLE
Is it possible to mutate websocket messages?

### DISCUSSION THREAD

**Discussion by gpadilha** [ory/oathkeeper#731] [Q&A] [answered]
Is it possible to use oathkeeper to mutate all websock messages using the mutator id_token?

I'm using the [@improbable-eng/grpc-web](https://github.com/improbable-eng/grpc-web) with the grpc-web-proxy to connect a vue app to a grpc api. However, the token header doesn't get to the api when the request is upgraded to websocket.

Thanks!

**Comment by aeneasr** [comment, marked as answer]
That's not possible at the moment unfortunately!

**Reply by gpadilha** [reply]
Thanks!
